bokeh/models/annotations.py

```diff
@@ -120,10 +120,24 @@ class Legend(Annotation):
     The %s for the legend background style.
     """)
 
+    inactive_props = Include(FillProps, help="""
+    The %s for the legend background style when inactive.
+    """)
+
+    hover_props = Include(FillProps, help="""
+    The %s for the legend background style when hovered.
+    """)
+
+    click_policy = Enum("hide", "mute")
```

Comment:
This can be documented better if it is a defined enum in the python enums module.